packages/expo-notifications/ios/EXNotifications/Notifications/Scheduling/SchedulerModule.swift

```diff
@@ -20,27 +20,23 @@ let calendarNotificationTriggerComponentsKey = "value"
 let calendarNotificationTriggerTimezoneKey = "timezone"
 // swiftlint:enable identifier_name
 
-public class SchedulerModule: Module {
+open class SchedulerModule: Module {
   public func definition() -> ModuleDefinition {
     Name("ExpoNotificationScheduler")
 
     AsyncFunction("getAllScheduledNotificationsAsync") { (promise: Promise) in
       UNUserNotificationCenter.current().getPendingNotificationRequests { (requests: [UNNotificationRequest]) in
```

Comment:
```suggestion
AsyncFunction("getAllScheduledNotificationsAsync") { 
     let request = await UNUserNotificationCenter.current().pendingNotificationRequests()
     return serializedNotificationRequests(requests)
```
`UNUserNotificationCenter` provides async compatible variants of most of these functions. We should use them where possible and remove any callback based apis